drivers/hmis/app/graphql/mutations/delete_client_file.rb

```diff
@@ -0,0 +1,23 @@
+module Mutations
+  class DeleteClientFile < BaseMutation
+    argument :file_id, ID, required: true
+
+    field :client, Types::HmisSchema::Client, null: true
```

Comment:
I think this should return the `file` as discussed, to be consistent with all other delete mutations. Do you think differently?